add-chain/testdata/superchain/configs/sepolia/expected_standard-candidate.yaml

```diff
@@ -0,0 +1,22 @@
+name: awesomechain_standard-candidate
+chain_id: 42069
+public_rpc: http://awe.some.rpc
+sequencer_rpc: http://awe.some.seq.rpc
+explorer: https://awesomescan.org
+
+superchain_level: 1
+superchain_time: null # Missing hardfork times are NOT yet inherited from superchain.yaml
+standard_chain_candidate: true
```

Comment:
I am wondering if we avoid polluting this file with this ephemeral information. 
Instead we could add an entry to a "candidates list" that we store somewhere (and do not export from the Go package).

Reply:
Decided against this in the end.